students/charlie_england/lesson01/assignment/inventory_management/inventoryClass.py

```diff
@@ -0,0 +1,17 @@
+# Inventory class
```

Comment:
It looks like this is a copy of the Inventory class. Any reason you have two of them?